egs/thchs30/s5/run.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+. ./cmd.sh ## You'll want to change cmd.sh to something that will work on your system.
+           ## This relates to the queue.
+. ./path.sh
+
+H=`pwd`  #exp home
+n=8      #parallel jobs
+
+thchs=/nfs/public/materials/data/thchs30-openslr
+#corpus and trans directory
+#you can obtain the database by uncommting the following lines
+#( cd $thchs
+#     echo "downloading THCHS30 at $PWD ..."
+#     wget http://www.openslr.org/resources/18/data_thchs30.tgz
+#     wget http://www.openslr.org/resources/18/resource.tgz
+#     tar xvf data_thchs30.tgz && tar xvf resource.tgz 
+#)
+
```

Comment:
The previous commented lines are not completely correct (after uncommenting) -- the $thchs variable will be either unset or set to come directory that won't probably be available anywhere else than your system. I'd say it would make more sense to write:

thchs=/nfs/public/materials/data/thchs30-openslr
if [ ! -d $thchs ] ; then
  thchs=./thchs30-openslr
  [ -d $thchs ] || mkdir $thchs
  cd $thchs
  echo "downloading THCHS30 at $PWD ..."
  wget http://www.openslr.org/resources/18/data_thchs30.tgz
  wget http://www.openslr.org/resources/18/resource.tgz
  tar xvf data_thchs30.tgz && tar xvf resource.tgz 
fi

eventually you can add some more checks to avoid the downloading and untaring the data each time it's  the recipe is run.